src/main/java/com/stripe/model/ExternalAccountTypeAdapterFactory.java

```diff
@@ -17,7 +17,7 @@
       return null; // this class only serializes 'ExternalAccount' and its subtypes
     }
 
-    final String SOURCE_OBJECT_PROP = "object";
+    final String sourceObjectProp = "object";
```

Comment:
don't we consider this a constant and then should follow the constant style of all caps? https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names